### PR TITLE
Add Brave Dungeon

### PR DESCRIPTION
**Pull request by bigger0gamer**
Adds Brave Dungeon.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/brave-dungeon/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Brave Dungeon"
+description = "RPG, Adventure - INSIDE SYSTEM\nJoin intrepid adventurer Al as she explores the dangerous labyrinths of Belfer Island in search of powerful Magic Items. Along the way Al will join forces with strong allies and face even stronger enemies. Can Al conquer the imposing dungeon known as Godshill and find the ultimate Magic Item?"
```
Comment (Leo626) [inline games/brave-dungeon/game.dat]: I don't really like the description from Nintendo's page. Use this one instead:
```
Brave Dungeon is an RPG developed and published by Inside System. A spin-off from The Legend of Dark Witch series on the 3DS. It's a dungeon crawler featuring the boss character Al as the main character with several other bosses from the franchise as playable characters.
```

File: games/brave-dungeon/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000001C0100"
+region = "usa"
+release_date = "2017-03-30"
+
+[[testcases]]
+title = "00040000001c0100"
```
Comment (Leo626) [inline games/brave-dungeon/game.dat]: For consistency the `c` should be capital like the one in `[[releases]]`.

File: games/brave-dungeon/game.dat [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+compatibility = "1"
+date = "2017-11-06"
+version = "HEAD-c6b2cc2"
+cpu = "Intel Core i5-7500"
+gpu = "GeForce GTX 1050"
```
Comment (Leo626) [inline games/brave-dungeon/game.dat]: `Nvidia GeForce GTX 1050`

**Comment by Leo626** [comment]
I'm pretty sure that's not the title screen of the game. The boxart for eShop games is the title screen. Also, you'll need to create the wiki page for this game. See the [Contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md) guide.

**Comment by TheKoopaKingdom** [comment]
@bigger0gamer Please address the review comments, or this PR will be closed.

**Pull request closed without merging**